src/Menu.tsx

```diff
@@ -589,6 +591,7 @@ const Menu = React.forwardRef<MenuRef, MenuProps>((props, ref) => {
             disabled={allVisible}
             internalPopupClose={len === 0}
             popupClassName={overflowedIndicatorPopupClassName}
+            popupOffset={overflowPopupOffset || [10, 15]}
           >
```

Comment:
Why is the default value this?

Reply:
i refer rc-menu docs's demo
if not, what value is appropriate

Reply:
@zombieJ @MadCcc @afc163 Pls take a look.